org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -1378,17 +1440,24 @@ public boolean load (String trxName)
 		try
 		{
 			pstmt = DB.prepareStatement(sql.toString(), m_trxName);	//	local trx only
-			for (int i = 0; i < m_IDs.length; i++)
+			if(m_isUUID)
```

Comment:
don't need the flag here, just pass in the uuid parameter and check that for null is simpler and safer